### PR TITLE
fix: import notion embed block

### LINKED ISSUE #4180: Support importing file embed from notion
Because the blocksuite has file block type.  So when importing notion, the file embed should transform to a file block instead of a link.

### PR DESCRIPTION
**Pull request by zuoxiaodong0815**
close #4180

### CODE REVIEW

**Review by fourdim — COMMENTED** [review]
image
Before state-transitions.png
After state-transitions.png.png

File: packages/blocks/src/__internal__/utils/types.ts [hunk 1]
```diff
@@ -247,6 +247,8 @@ export type SerializedBlock = {
   children: SerializedBlock[];
   sourceId?: string;
   caption?: string;
+  name?: string;
```
Comment (doodlewind) [inline packages/blocks/src/__internal__/utils/types.ts:250]: Are we going to deprecate using this type? /cc @Saul-Mirone @fourdim

**Review by doodlewind — APPROVED** [review]

File: packages/blocks/src/__internal__/utils/types.ts (hunk 1 above)
Comment (zuoxiaodong0815) [reply packages/blocks/src/__internal__/utils/types.ts:250]: Ok. Because the attachment block has the name property.

**Comment by zuoxiaodong0815** [comment]
> image Before state-transitions.png After state-transitions.png.png

the fix pr is here. https://github.com/toeverything/blocksuite/pull/4587

File: packages/blocks/src/__internal__/utils/types.ts (hunk 1 above)
Comment (Saul-Mirone) [reply packages/blocks/src/__internal__/utils/types.ts:250]: > Are we going to deprecate using this type? /cc @Saul-Mirone @fourdim

Yep

**Pull request merged**